internal/services/databricks/databricks_workspace_resource.go

```diff
@@ -1099,3 +1180,84 @@ func workspaceCustomParametersString() []string {
 		"custom_parameters.0.vnet_address_prefix",
 	}
 }
+
+func flattenWorkspaceEnhancedSecurity(input *workspaces.EnhancedSecurityComplianceDefinition) []interface{} {
+	if input == nil {
+		return nil
+	}
+
+	enhancedSecurityCompliance := make(map[string]interface{})
+
+	if v := input.AutomaticClusterUpdate; v != nil {
+		enhancedSecurityCompliance["automatic_cluster_update_enabled"] = *v.Value != workspaces.AutomaticClusterUpdateValueDisabled
```

Comment:
`Value` is a pointer, so this should be nil checked to prevent a crash
```suggestion
		enhancedSecurityCompliance["automatic_cluster_update_enabled"] = pointer.From(v.Value) != workspaces.AutomaticClusterUpdateValueDisabled
```